src/layout/Grid.js

```diff
@@ -87,7 +87,9 @@
         return this;
     };
 
-    Grid.prototype.getCell = function (row, col) {
+    Grid.prototype.getCell = function (_row, _col) {
+        var row = _row * this.cellDensity();
+        var col = _col * this.cellDensity();
```

Comment:
That means that setContent(5, 5, widget) is not symmetrical with a getCell(5, 5)?